### PR TITLE
Bugfix #151

### PR DESCRIPTION
**Pull request by srikanth716**

### CODE REVIEW

**Review by gsasikumar — COMMENTED** [review]
.env is a local file. why is it even checked in. Delete this file. Update the read me with a sample .env file

File: .github/workflows/android-custom-build.yml [hunk 1]
```diff
@@ -1,15 +1,15 @@
 name: ID PASS - MOSIP Resident Application Custom build
 
 env:
-  backendServiceDefaultUrl: https://api-internal.qa4.mosip.net/residentmobileapp
```
Comment (gsasikumar) [inline .github/workflows/android-custom-build.yml:4]: why is it hard coded in the yml? it should be an input for yml. Look at this https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/ 
Work with chandra and his team if you need clarity.

File: .github/workflows/android-custom-build.yml [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+  backendServiceDefaultUrl: https://api.qa-121.mosip.net/residentmobileapp
 
 on:
   workflow_dispatch:
     inputs:
       backendServiceUrl:
         description: 'Backend service URL'
         required: true
-        default: 'https://api-internal.qa4.mosip.net/residentmobileapp'
```
Comment (gsasikumar) [inline .github/workflows/android-custom-build.yml:12]: same as earlier.

**Review by gsasikumar — CHANGES_REQUESTED** [review]

**Comment by srikanth716** [comment]
Closing this PR as the changes are already included

**Pull request closed without merging**